src/cache/cache_las.c

```diff
@@ -724,8 +715,7 @@ __wt_las_insert_block(WT_SESSION_IMPL *session, WT_CURSOR *cursor,
 			 * table. (We check the length because row-store doesn't
 			 * write zero-length data items.)
 			 */
-			if (multi->page_las.las_skew_newest &&
-			    upd == list->onpage_upd &&
+			if (upd == list->onpage_upd &&
```

Comment:
@agorrod, @keithbostic this change in this ticket was not required for correctness -- I just couldn't work out why we were only using birthmarks when skewing newest.  If we make any progress reproducing WT-4144, reverting this change would be the first thing I'd try.